.github/workflows/potential-duplicates.yml

```
@@ -0,0 +1,36 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+jobs:
```

Minor, but for consistency with the other Workflow files could you insert a newline here :smile: 

```suggestion
    types: [opened, edited]

jobs:
```